db/change_cache_test.go

```diff
@@ -1299,44 +1299,6 @@ func TestChannelRace(t *testing.T) {
 	changesCtxCancel()
 }
 
-// Test that housekeeping goroutines get terminated when change cache is stopped
-func TestStopChangeCache(t *testing.T) {
```

Comment:
This seems like a valid test case - do you think this scenario has coverage elsewhere?

Reply:
I think that the test code can be left in, but it wasn't using the leaky tap feed. I am happy to do that.